### PR TITLE
The gallery is now build automatically

### PR DESCRIPTION
**Pull request by NelleV**
@choldgraf This is going to break a bunch of stuff :D 
also, plot_labels takes way too long. We'll have to fix this.

### CODE REVIEW

**Comment by choldgraf** [comment]
lol I like how our new system of auto-merging consists of a bunch of PRs that say "THIS WILL DEFINITELY BREAK EVERYTHING"

File: doc/index.rst [hunk 1]
```diff
@@ -10,6 +10,7 @@ Welcome to Watchtower's documentation!
    :maxdepth: 2
    :caption: Contents:
 
+  auto_examples/index
```
Comment (choldgraf) [inline doc/index.rst]: Do we want to call it "auto_examples"? Maybe something better?

Comment (choldgraf) [inline doc/index.rst]: what about "a_e"?

Comment (NelleV) [inline doc/index.rst]: I think that's a bit too long…

Comment (choldgraf) [inline doc/index.rst]: can we name the folder `partyparrot`?

**Pull request merged**